### DISCUSSION TITLE
RSS feed reader + What is the Scope of Zen Browser?

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#769] [Ideas]
### What feature would you like to see?

One neat feature that Vivaldi has is a [feed reader](https://vivaldi.com/features/feed-reader). It implements feed auto-discovery and shows a little feed button in the URL bar if a feed is detected on the page, and also has a built-in feed reader directly in the browser. Even though I said "RSS" in the title, this feature would ideally also support Atom and JSON feeds.

### Other details

I'd also like to ask: what is the scope of Zen Browser? What features should/can be requested? A built-in feed reader is a pretty atypical feature present only in Vivaldi (from my knowledge), and it has proved immensely practical for me (though it could do with some love, it using the mail interface feels quite clunky). Would that be something this project would like to implement?
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/677_

**Comment by 5onderling** [comment]
I personally don't need or want an RSS feed reader in my browser but I would very much like an RSS feed detector. I used the Orion browser for a while and it has such a feature. It finds all RSS (or Atom or JSON) feeds on a page and gives the option to copy the url or add the feed to your local RSS reader application (NetNewsWire in my case). Sadly I couldn't find an extension that does exactly that (properly) but I might consider doing it myself (I hate Firefox extension development though).
<img width="568" height="226" alt="image" src="https://github.com/user-attachments/assets/3ba48d52-7c42-4f68-8461-8b12960fca97" />
